cli.sh

```diff
@@ -1270,7 +1323,7 @@ git hooks update [enable|disable]
 
     if [ "$1" != "force" ]; then
         if ! is_update_available; then
-            echo "  Githooks is already on the latest version"
+            echo "  Current Githooks version: $CURRENT_VERSION is newer than the remote version: $LATEST_VERSION"
```

Comment:
Can we revert this change? I don't think the extra information is useful for most people.

Reply:
Yeah, I think it's useful when working on update related changes, but not necessarily that useful otherwise.